src/charts/heatmap/heatmap.directive.js

```diff
@@ -144,25 +154,36 @@ angular.module('patternfly.charts').directive('pfHeatmap', function ($compile, $
       showLegend: '=?',
       legendLabels: '=?',
       maxBlockSize: '@',
+      minBlockSize: '@',
       blockPadding: '@',
       thresholds: '=?',
       heatmapColorPattern: '=?',
-      clickAction: '=?'
+      clickAction: '=?',
+      rangeOnHover: '=?',
+      rangeHoverSize: '@',
+      rangeTooltips: '=?'
     },
     templateUrl: 'charts/heatmap/heatmap.html',
     controller: function ($scope) {
       var thresholdDefaults = [0.7, 0.8, 0.9];
       var heatmapColorPatternDefaults = ['#d4f0fa', '#F9D67A', '#EC7A08', '#CE0000'];
       var legendLabelDefaults = ['< 70%', '70-80%' ,'80-90%', '> 90%'];
+      var rangeTooltipDefaults = ['< 70%', '70-80%' ,'80-90%', '> 90%'];
       var heightDefault = 200;
 
       //Allow overriding of defaults
+      if ($scope.minBlockSize === undefined || isNaN($scope.minBlockSize)) {
+        $scope.minSize = 5;
```

Comment:
I think it is OK to have a max range of 5-50 and still set the min to 2. I believe we should follow the spec regardless of range hover being enabled.